### ISSUE TITLE
Source / Live Preview modes support

### ISSUE THREAD

**Issue by LilaRest** [svonjoi/obsidian-heading-level-indent#3]
Hi ! Thanks for this awesome plugin :blush: 

I just wanted to know if you planed to work soon on Read / Live Preview modes support or if it is more like a long term goal in the Roadmap ?

**Comment by svonjoi** [comment]
thanks for your feedback =)
Read mode (preview mode) is already supported, also it's the only thing thats works
I plan to work on source/live preview mode soon, also fix the delay in read mode

**Comment by LilaRest** [comment]
Yup sorry, I had quickly written this issue, I was indeed talking about Source / Live Preview modes support.

Happy to hear that ! This plugin is a must have IMO.

I'm actually indenting using CSS, which doesn't allow to indent content (only headings).

**Comment by svonjoi** [comment]
glad to see it's useful for already some people :3

**Comment by MarkMcElroyDotCom** [comment]
Signed up for GitHub just to say thank you for the work you're doing on this plug-in. Ever since I started using Obsidian (and other, similar apps), I've wanted to indent content under headings in exactly the way this plug-in does. 

Please do continue work on the plug-in! I appreciate your work very much.

**Comment by svonjoi** [comment]
> Signed up for GitHub just to say thank you for the work you're doing on this plug-in. Ever since I started using Obsidian (and other, similar apps), I've wanted to indent content under headings in exactly the way this plug-in does.
> 
> Please do continue work on the plug-in! I appreciate your work very much.

Thanks =)

**Comment by tonyjly** [comment]
Thanks for creating this plugin @svonjoi. +1 for the feature request

**Comment by Finni123** [comment]
+1 for this feature, I love the plugin but 99% I work in Live Preview mode, so I would be super happy if support for that would be added

**Comment by geeemok** [comment]
+1 for thanks
+1 for this feature

**Comment by russarr** [comment]
Thanks from me , too . I vote for this feature

**Comment by elvarb** [comment]
FYI there is a css snipped that does this in preview mode. It can be very resource heavy on big notes.

https://github.com/zamsyt/obsidian-snippets/wiki/Indent-headings

**Comment by svonjoi** [comment]
> FYI there is a css snipped that does this in preview mode. It can be very resource heavy on big notes.
> 
> https://github.com/zamsyt/obsidian-snippets/wiki/Indent-headings

This CSS applies the same indentation to both the heading and its content. I did not find the way to solve this via CSS so I've created the plug

**Comment by burninester** [comment]
> > FYI there is a css snipped that does this in preview mode. It can be very resource heavy on big notes.
> > https://github.com/zamsyt/obsidian-snippets/wiki/Indent-headings
> 
> This CSS applies the same indentation to both the heading and its content. I did not find the way to solve this via CSS

they updated it, it indents the text underneath the header too now: https://github.com/zamsyt/obsidian-snippets/issues/15

**Comment by svonjoi** [comment]
> they updated it, it indents the text underneath the header too now: [zamsyt/obsidian-snippets#15](https://github.com/zamsyt/obsidian-snippets/issues/15)
> 
> dont know why the issue is still open on zamsyt's repo, but it's working for me

I didn't pay attention to it. this css snippet works well so i will archive this repo and use this css snippet when i figure out how to modify it to my needs

**Comment by svonjoi** [comment]
#26
